railties/lib/rails/commands/server/server_command.rb

```diff
@@ -9,7 +9,7 @@
 require "rails/dev_caching"
 
 module Rails
-  class Server < ::Rack::Server
+  class Server < ::Rack::Server #:nodoc:
```

Comment:
This could technically be a backward incompatible change, but:

1. I don't think we should list `Rails::Server` in the documentation at all.
2. This change aims Rails 6, so it should be okay.

Reply:
1. We have listed it in the past, so it's technically public API.
2. Just because it's a major doesn't mean we should just make undue backwards incompatible changes willy-nilly.

I am learning towards the Rails::Server published as public API being a mistake and not being intended. I don't think any of the other Railties command helpers are public.

What does a GitHub search for `Rails::Server` say? It might corroborate the above 😊

Reply:
If we find there's no real usage of `Rails::Server` on GitHub, I'm 👍 on `# :nodoc:` (but do add the space so it matches the command nodoc 🤓)